### DISCUSSION TITLE
IBC Workshop Summary

### DISCUSSION THREAD

**Discussion by schuettpelze** [pteridogroup/ppg#89] [Announcements]
Dear PPG,

As you know, a PPG workshop was held at the International Botanical Congress in Madrid this past July. In all, about 60 pteridologists were able to attend, representing 22 countries. Over the course of a few hours, a wide variety of topics were discussed (many thanks to Emily Sessa for taking notes). Here, we’ve listed the highlights.

Phase 1 (genus-level classification):

- Schuettpelz provided a recap of PPG I and a summary of the PPG II approach and phase 1 progress to date (https://docs.google.com/presentation/d/1e_3H3UoYFlwfnUKbatGWSjnh3O08lN5hyZe_UAZbEOE/edit?usp=sharing). He then facilitated a phase 1 discussion.
- It was noted that the focus to date has been proposal by proposal and we haven’t stepped back to look at the whole of what’s emerging. Are we happy where things are headed overall? Are there areas that are out of sync with each other? Should we make an attempt to harmonize things?
- PPG I has been criticized by some for being overly splitty. Are we headed there again and, if so, is that acceptable? Of course, this problem is not unique to pteridophytes and many issues are related to rank rather than monophyly. Perhaps an (eventual) attempt to morphologically delineate genera will help, by establishing which genera are well-defined versus fuzzy at the boundaries? Fundamental is whether or not there is a good reason, phylogenetically and morphologically, for what is being proposed. Of note, not everything proposed has passed, and some of the more extreme cases of splitting were rejected. 
- The process we are using is iterative and continuous (until we determine a completion date for PPG II). Just because a proposal passes, does not mean it is the final word. Someone else can submit a new proposal involving the taxa the classification that involves those taxa. In some cases, there have been alternative proposals that are incompatible and, in at least one case, both were rejected because there wasn’t a simple vote for/against. Need a better option for resolving these cases (it was difficult to foresee all the possible situations). Moving forward, will make it explicit that a proposal has to be for one specific change. Leaving things stand in a state of ambivalence is also an option, should the community choose to accept it.
- Nomenclature stability is important. We should avoid going back and forth between splitting and lumping. Even though the ranks are arbitrary, some have practical consequences and are more important than others. We should try to keep the genus level as stable as possible. Others noted that sometimes there is no other option and leaving things as they are is the worst outcome.
- Is there a case to be made for accepting paraphyly in some situations? If there is a real chance that more data and more work would resolve a situation, that might be an argument in favor of  being conservative. Could we mark taxa as in need of further work?
- Some suggested we must be pragmatic. For people doing conservation, wetland delineation, etc., there are consequences when we make changes. Teaching also becomes difficult. What does it mean if we can’t identify taxa morphologically, even if evolutionary you can delineate them? Others noted that the notion that larger genera are more practical in ecology is problematic, as this is the least ecologically informative solution. Big and stable is not always the best.
- What journal for PPG II? When thinking about where to submit our manuscript, we need to remember we are an international community and need to be mindful of journal policies. There was a recommendation for Taxon, where this type of paper would be very welcome. Others agreed, as Smith et al. (2006) was there and there is thus history. Could put this (and the like) up as an idea “proposal” for a vote?
- What about format? Same as PPG I? A key would be nice, albeit not necessary. The exercise of building a key could highlight problematic areas. However, one institution pitched a project for students to do this and none applied! Many liked the PPG I format, including the species counts. We could link to the FTOL tree and improve viewer/pruning?
- Finding consensus. Some noted that new folks want to be involved, but need to be informed. We don’t want a situation where people vote and then afterwards go and read about what that issue was actually about. Several noted that they only voted when they had time to read the papers and make a decision.
- Would it be useful to incorporate some kind of consensus metric? How do we quantify confidence? Note that we typically get about 60-80 votes out of our group of nearly 200, so we’re getting less than 50% participation.

Phase 2 (species-level classification):

- Nitta provided an overview of the phase 2 approach (https://joelnitta.github.io/2024-07-24-ppg/) and facilitated a discussion. One major change coming in phase 2 is that we will be using the [Rhakhis](https://rhakhis.rbge.info/) database editor system of [World Flora Online (WFO)](https://worldfloraonline.org/) to maintain the taxonomic database. This goes along with the planned integration of [PPG into WFO as a Taxonomic Expert Network (TEN)](https://about.worldfloraonline.org/tens/pteridophyte-phylogeny-group).
- It was suggested that we need a mechanism (like the github discussion page) to facilitate discussion within the genus/family groups/committees. This may be especially important for large lineages.
- Someone noted the need for a means of marking species as good versus in need of attention. How might we establish a threshold for saying “good enough” or only provisional?
- Another participant suggested a poll to determine who wants to participate and where.
- Some concern was raised about the time needed to recreate a species-level classification in WFO. However, having it there (and editable via Rhakhis) will be a big step forward, as it opens the door to all kinds of gradual improvements. Longevity is also a consideration.
- Biggest feedback on PPG from Australia was not being able to place species. A project to find the types and place them in the classification took four years. This would have been much easier with a species-level resource and until that is in place, the PPG classification will not have the uptake or utility we strive for. 
- It was noted that not all groups will need work and in some cases it might be appropriate for working groups to simply sign off on the existing classification (i.e., [Michael Hassler’s World Ferns](https://www.worldplants.de/world-ferns/ferns-and-lycophytes-list)). At the same time, it may be necessary to leave some difficult problems for later.
- While some individuals are taxonomically focused, others are local, regional experts. We need to think about  how to involve them. An idea that may work is to have the taxonomists make the first pass, then geographic specialists look at it and make suggestions/corrections.
- Is there a way to seek funding for our work? Could next-generation digitization (e.g., focusing on spores, scales, and other microscopic features) of types be an angle? Perhaps there is an opportunity for broader impacts, including student training in the various taxonomic actions involved. How to do it, what all the terminology means, what you have to consider, etc.
- Some suggested it might prove too difficult to account for all synonyms. However, within WFO, we’ll need to deal with all names.
- Do we want to publish a list of accepted names? Or is it enough to contribute to the update and regular maintenance of a globally accessible database?
- It was noted that the classification currently presented in WFO does not align with PPG. At the moment, WFO pulls fern data from Kew’s [World Checklist of Vascular Plants](https://powo.science.kew.org/about-wcvp) (WCVP) and a first step will be to replace the WFO data with an updated version (using World Ferns as a starting point). Work is underway by Nitta with the help of WFO staff to import these data, integrating changes already approved via voting.
- We need to establish a standard workflow for clade-focused committees. Part of this might involve sharing updates with World Ferns and IPNI.
- Need to consider time required and timeline. What is more urgent, publishing PPG2 without everything being finished, or planning for PPG3? How far do we want to go, and what’s the estimate for how long the species level work will take to get done? It was noted that PPG2 will only be genus level and above. 

We would welcome further discussion on any of the above topics. To keep things organized and focused, perhaps it would be simplest to create a new discussion for each topic you wish to discuss.

All the best,
Eric and Joel

**Comment by BlancaRFerns** [comment]
Thanks Eric, Joel and Emily for providing a summary of trends in PPG-II. I have been back and forth in previous submitted proposals to resolve where we as pteridologists stand for expressing monophyly in several split genera. I agree with the philosophy of PPG-I continuing for PPG-II. Those families with recent phylogenies and mostly Old World like Athyriaceae, Thelypteridaceae, Polypodiaceae (mostly Grammitidoideae) include most of those passed submissions. But, among non-passed submissions we need to be aware of the persistence of these alternative classifications for families and genera that are and will be used in recognizing new taxa. Saying this brings me to where to publish the new proposal of PPG-II that has to be easily available. Taxon offers OA and many of us are members of IAPT.  American Fern Journal is indeed a possibility, but I am unsure of the easy access for non-BioOne subscribers.

**Comment by Meza-Torres** [comment]
Dear PPG,
I tried to open a discussion about where to post PPG II (I don't know if I did it right). The discussion is here:
https://github.com/pteridogroup/ppg/discussions/90
